weaver/cli.py

```diff
@@ -1173,9 +1174,12 @@ def jobs(self,
              sort=None,             # type: Optional[Sort]
              page=None,             # type: Optional[int]
              limit=None,            # type: Optional[int]
-             status=None,           # type: Optional[StatusType]
+             status=None,           # type: Optional[Union[StatusType, List[StatusType]]]
              detail=False,          # type: bool
              groups=False,          # type: bool
+             process=None,          # type: Optional[str]
+             provider=None,         # type: Optional[str]
+             tags=None,             # type: Optional[Union[str, List[str]]]
```

Comment:
These args are added, but they seem absent from the CLI options (as well as status)